docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst

```diff
@@ -25,3 +27,115 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
 
 .. todo::
    Describe cause model
+
+Data Descriptions
+-----------------
+
+.. list-table:: State Definitions
+	:widths: 5 10 10
+	:header-rows: 1
+	
+	* - State
+	  - State name
+	  - Definition
+	* - C
+	  - With **C**\ ondition
+	  - Simulant was born with haemolytic disease or other neonatal jaundice
+	* - F
+	  - **F**\ ree of condition
+	  - Simulant was not born with haemolytic disease or other neonatal jaundice
+	  
+.. list-table:: State Data
+	:widths: 5 10 10 20
+	:header-rows: 1
+	
+	* - State
+	  - Measure
+	  - Value
+	  - Notes
+	* - C
+	  - prevalence
+	  - prevalence_c384
+	  -
+	* - C
+	  - birth prevalence
+	  - birth_prevalence_c384
+	  - 
+	* - C
+	  - excess mortality rate
+	  - :math:`\frac{\text{deaths_c384}}{\text{population} \,\times\, \text{prevalence_c384}}`
+	  -
+	* - C
+	  - disability weight
+	  - :math:`\displaystyle{\sum_{s\in \text{sequelae_c384}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+	  -
+	* - F
+	  - prevalence
+	  - 1-prevalence_c384
+	  -
+	* - F
+	  - birth prevalence
+	  - 1-birth_prevalence_c384
+	  - 
+	* - F
+	  - excess mortality rate
+	  - 0
+	  -
+	* - F
+	  - disability weight
+	  - 0
+	  -
+	* - All
+	  - cause-specific mortality rate
+	  - :math:`\frac{\text{deaths_c384}}{\text{population}}`
+	  -
+	 
+	  
+.. list-table:: Transition Data
+	:widths: 10 10 10 10 10
+	:header-rows: 1
+	
+	* - Transition
+	  - Source State
+	  - Sink State
+	  - Value
+	  - Notes
+	* - N/A
+	  - N/A
+	  - N/A
+	  - N/A
+	  - N/A
+	  
+.. list-table:: Data Sources and Definitions
+	:widths: 10 10 20 20
+	:header-rows: 1
+	
+	* - Variable
+	  - Source
+	  - Description
+	  - Notes
+	* - prevalence_c384
+	  - como
+	  - Prevalence of hemolytic disease and other neonatal jaundice
+	  - 
+	* - birth_prevalence_c384
+	  - como
+	  - Proportion of babies born with hemolytic disease and other neonatal jaundice
+	  - age_group_id = 164 and measure = 6 (incidence)
+	* - deaths_c384
+	  - codcorrect
+	  - Count of deaths due to hemolytic diseases and other neonatal jaundice
+	  - 
+	* - population
+	  - Demography
```

Comment:
Maybe let's just standardize and make everything lower case, i.e. "demography"?  I'll edit my table as well.